<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kalkulačka</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>341</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="3">
      <widget class="QPushButton" name="Mul">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>×</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="Minus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="Number3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="Number4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="Power">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>x^n</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="Number1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="Number7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="Root">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>n√x</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="Idiv">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>idiv</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="Left_bracket">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="Number5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="Number8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="Equals">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="Backspace">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>←</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="Div">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>÷</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="Number6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="Plus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="Fact">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>x!</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="Number0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="Number9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="Dot">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="Number2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="Right_bracket">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="Clear">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #284b63;
	padding: 10px;
	color: white;
	border-style: outset;
    border-width: 1px;
    border-radius: 15px;
    border-color: #b8cbd8;
}

QPushButton:pressed {
	background-color: #b8cbd8;
	border: 1px	solid #b8cbd9;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>6</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="Display">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="Rest_label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: white;
}</string>
       </property>
       <property name="text">
        <string>zbytek:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Rest_of_number">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>362</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="Options">
    <property name="title">
     <string>Možnosti</string>
    </property>
    <addaction name="Help"/>
   </widget>
   <addaction name="Options"/>
  </widget>
  <action name="Help">
   <property name="text">
    <string>Nápověda</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
